File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(number int) string {
	ret := ""
    if number % 3 == 0 {
        ret = ret + "Pling"
    }
	if number % 5 == 0 {
        ret = ret + "Plang"
    }
	if number % 7 == 0 {
        ret = ret + "Plong"
    }
	if ret == "" {
        return strconv.Itoa(number)
    } else {
    	return ret
    }
}

File: r/raindrops/raindrops.R
raindrops <- function(number) {
  output <- ""
  if (number %% 3 == 0) {
    output <- "Pling"
  }
  if (number %% 5 == 0) {
    output <- paste0(output, "Plang")
  }
  if (number %% 7 == 0) {
    output <- paste0(output, "Plong")
  }
  if (output == "") {
    output <- as.character(number)
  }
  return(output)
}
